util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -248,8 +248,10 @@ macro (seqan_build_system_init)
     ## options
 
     # SeqAn Version Check
-    if (NOT SEQAN_VERSION_CHECK)
-        set (SEQAN_DEFINITIONS "${SEQAN_DEFINITIONS};-DSEQAN_DISABLE_VERSION_CHECK")
+    if ("${SEQAN_VERSION_CHECK}" STREQUAL "disable")  # Disable completely
+        set (SEQAN_DEFINITIONS ${SEQAN_DEFINITIONS} -DSEQAN_DISABLE_VERSION_CHECK)
+    elseif ("${SEQAN_VERSION_CHECK}" STREQUAL "opt-in")  # Build it but make it opt-in
+        set (SEQAN_DEFINITIONS ${SEQAN_DEFINITIONS} -DSEQAN_VERSION_CHECK_OPT_IN)
     endif ()
```

Comment:
Here there seems to be a single three-variate variable at cmake level, but then two variables at c-macro-level